src/realm/parser/driver.cpp

```diff
@@ -1691,6 +1698,16 @@ auto ParserDriver::column(LinkChain& link_chain, std::string identifier) -> Sube
         util::format("'%1' has no property '%2'", link_chain.get_current_table()->get_class_name(), identifier));
 }
 
+auto ParserDriver::dictionary_column(LinkChain& link_chain, std::string identifier) -> SubexprPtr
```

Comment:
What's the point of passing`identifier` by value and assign it at line 1703?

Reply:
As you can see, it is both input and output of the translate function.

Reply:
Yes, my point is that probably would have made more sense to pass `identifier` by reference. But it is ok.